Cloud SIEM content release notes for March 24 2025 (#5201)
SumoLogic/sumologic-documentation: blog-cse/2025-03-24-content.md

---
title: March 24, 2025 - Content Release
image: https://help.sumologic.com/img/sumo-square.png
keywords:
  - rules
hide_table_of_contents: true    
---

import useBaseUrl from '@docusaurus/useBaseUrl';

> This content release includes Threat Intelligence match rules utilizing the new [hasThreatMatch](/docs/cse/rules/cse-rules-syntax/#hasthreatmatch) operator to work with global and custom threat intelligence feeds.

Reviewer: ```suggestion
This content release includes Threat Intelligence match rules that use the new [`hasThreatMatch`](/docs/cse/rules/cse-rules-syntax/#hasthreatmatch) operator to support both global and custom threat intelligence feeds.
```

> To reduce initial signal volume, basic inbound and outbound IP address threat match rules with a low or medium confidence level are disabled by default (see below). It is highly recommended that tuning be applied to these rules prior to enabling them to reduce signal volume, and therefore entity risk assignment, to manageable levels.

Reviewer: ```suggestion
To reduce initial signal volume, basic inbound and outbound IP address threat match rules with a low or medium confidence levels are disabled by default (see below). We strongly recommend tuning these rules before enabling them to keep signal volume, and therefore entity risk assignment, to manageable levels.
```
